### PR TITLE
feat: add support for Automatically deploy Adaptive Protection suggested rules

### LINKED ISSUE #60: Add support for Automatically deploy Adaptive Protection suggested rules
Add support for Automatically deploy Adaptive Protection suggested rules

https://cloud.google.com/armor/docs/adaptive-protection-auto-deploy

### PR DESCRIPTION
**Pull request by imrannayer**
Fix #60

### CODE REVIEW

File: examples/security-policy-all/main.tf [hunk 1]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+
```
Comment (g-awmalik) [inline examples/security-policy-all/main.tf:17]: nit: whitespace

File: examples/security-policy-managed-protection-plus/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Cloud Armor Policy with rules supported by [Cloud Armor Managed Protection Plus (CAMP+)](https://cloud.google.com/armor/docs/managed-protection-overview)
+
+This example configures a single cloud armor policy with different types of rules.
```
Comment (g-awmalik) [inline examples/security-policy-managed-protection-plus/README.md:3]: Is there a list of rules we can reference here?

**Review by g-awmalik — APPROVED** [review]
LGTM. Few nits.

File: examples/security-policy-managed-protection-plus/versions.tf
[the remark this answers is not in the document]
Comment (imrannayer) [reply examples/security-policy-managed-protection-plus/versions.tf:1]: I think it is a good idea to remove it. But as of now almost all the examples have version.tf

**Pull request merged**